### PR TITLE
readme updated, run.sh added

### PR DESCRIPTION
**Pull request by jamesturner-dev**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,9 +1,11 @@
+docker run --rm -d -p 6379:6379 --name redis  -v $PWD/data/redis:/data redis
+
+sleep 2
+
 . ./metatron.env
 
 export FLASK_APP=vault
 export FLASK_ENV=development
 export SECRET_KEY='scully:sw33tp0tat0'
```
Comment (macterra) [inline run.sh:9]: Not sure the `SECRET_KEY` is necessary. Can we remove it? If not, we should probably move it into metatron.env

**Review by macterra — CHANGES_REQUESTED** [review]

File: run.sh (hunk 1 above)
Comment (aiggdev) [reply run.sh:9]: It's just the 2 previous sh scripts combined that already existed.

Comment (macterra) [reply run.sh:9]: True but the question stands. Can we get rid of the key or should we move it?

Comment (macterra) [reply run.sh:9]: Please move `SECRET_KEY` to the env file for now

Comment (macterra) [reply run.sh:9]: OK I will do it

**Pull request merged**